### PR TITLE
docs: Email Group Option in Email Campaign

### PR DESCRIPTION
**Pull request by abhishekbalam**
ERPNext PR:
https://github.com/frappe/erpnext/pull/22731

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ Then the emails sent would look like this:
 Status indicates the state of the Email Campaign, the various Statuses are:
 
 - **Scheduled**: When the Email Campaign has not yet started but scheduled on a future 'Start Date'.
-- **In Progress**: When there are still more campaign mails to be sent.
-- **Completed**: When all the emails have been sent according to the schedule set up in the Campaign.
+- **In Progress**: When the Email Campaign is in progress.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md:80]: The previous statement is more explanatory. Of course, the user would understand that 'In progress' for an Email campaign would mean that the campaign is in progress. But it means that some emails are sent and some are left to be sent.

File: erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md [hunk 2]
```diff
@@ -77,8 +77,8 @@ Then the emails sent would look like this:
...
+- **Completed**: When the Email Campaign is completed.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md:81]: Same as above. Please change it back to the previous explanation.

File: erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md (hunk 1 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/CRM/email-campaign.md]: Made the changes discussed

**Comment by abhishekbalam** [comment]
@akshitakl 
Made the changes, sorry for the delay.

**Pull request merged**